_posts/2022-11-14-release-2022.11.14.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Release Day 2022.11.14: Interface Overhaul"
+description: "Today is release day!"
+published: 2022-11-14
+layout: post
+thumb: https://raw.githubusercontent.com/bottlesdevs/data/main/releases/2022.10.14/release-day.png
+background: https://raw.githubusercontent.com/bottlesdevs/data/main/backgrounds/2022.10.14.png
+published: true
+comments: true
+is_release_day: true
+is_release_day_color: '#1699e7'
+user_nick: cyberphantom52
+user_name: Inam Ul Haq (cyberphantom52)
+user_tag: Contributor
+co_user_nick: TheEvilSkeleton
+co_user_name: Hari Rana (TheEvilSkeleton)
+co_user_tag: Developer
+---
+
+It’s Release Day again!
+
+A lot of work has been done this time around and it’s time to celebrate another release day!
+
+This release comes with a redesign of the details and settings page. Also the programs page has been removed.
+
+## News
+Ok now let’s get to the point. What’s new in this version?
+
+### Brand new details view
+The details view has been completely redesigned to make Bottles more mobile friendly. The sidebar has been removed and the pages have been moved into the details view itself.
+
+The option to run an executable is now a nice visible pill to bring it into the focus and launch options are now neatly placed in their own popover next to it. There is a new bottom bar below the programs list to add/install new programs.
+
+<div class="grid-pics">
+    <img tooltip="Details View" class="on-light" src="/uploads/details-view-light.png" />
```

Comment:
This image doesn't work. The file path is `/uploads/details-view.png` (but I suggest renaming to `details-view-light.png`).

Also, the image is out of focus for both light and dark styles.